.github/workflows/cts.yml

```diff
@@ -0,0 +1,86 @@
+name: CTS
+
+on:
+  pull_request:
+    types: [labeled, opened, synchronize]
+  workflow_dispatch:
+
+env:
+  CARGO_INCREMENTAL: false
+  CARGO_TERM_COLOR: always
+  RUST_BACKTRACE: full
+  MSRV: "1.84"
+
+jobs:
+  cts:
+    # For pull requests, run only run if we add the "PR: run CTS" label
+    if: "github.event_name == 'workflow_dispatch' || (github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'PR: run CTS'))"
```

Comment:
I tried a bunch of versions of this, and GitHub kept saying it was an error in the yaml syntax. But I'm not sure if this one is correct, or if it's interpreted as a non-null string that will always be true.

Reply:
By testing with and without the `PR: run CTS` label, I confirmed this is working.